vendor/vendor.json

```diff
@@ -29,101 +43,130 @@
 			"revisionTime": "2016-01-05T22:08:40Z"
 		},
 		{
+			"checksumSHA1": "bKMZjd2wPw13VwoE7mBeSv5djFA=",
 			"path": "github.com/matttproud/golang_protobuf_extensions/pbutil",
-			"revision": "d0c3fe89de86839aecf2e0579c40ba3bb336a453",
-			"revisionTime": "2015-10-11T12:25:29+02:00"
+			"revision": "c12348ce28de40eed0136aa2b644d0ee0650e56c",
+			"revisionTime": "2016-04-24T11:30:07Z"
 		},
 		{
+			"checksumSHA1": "GGb6UcrDCNMWWIjyjDgvauGw9qI=",
 			"path": "github.com/prometheus/client_golang/prometheus",
-			"revision": "18acf9993a863f4c4b40612e19cdd243e7c86831",
-			"revisionTime": "2016-02-23T13:27:30+01:00"
+			"revision": "e56081f7b9d0c522b3b3b3ef154be6d74001382f",
```

Comment:
For consistency, I'd prefer this to be set to v0.8.0 AKA c5b7fccd204277076155f10851dad72b76a49317